mso/resource_mso_schema_site.go

```diff
@@ -106,21 +108,31 @@ func resourceMSOSchemaSiteImport(d *schema.ResourceData, m interface{}) ([]*sche
 		apiSiteId := models.StripQuotes(tempCont.S("siteId").String())
 		apiTemplate := models.StripQuotes(tempCont.S("templateName").String())
 
-		if apiSiteId == stateSiteId {
+		if apiSiteId == stateSiteId && apiTemplate == template {
+			// d.SetId(fmt.Sprintf("%s/sites/%s-%s", schemaId, stateSiteId, template))
```

Comment:
Should the commented out code be removed?

Reply:
actually left the comment there on purpose that this is the id we would actually want, but I did not want to introduce a breaking change into the resource.